scripts/install-mender.sh

```diff
@@ -83,7 +108,25 @@ parse_args() {
                 fi
                 ;;
             --demo)
-                SELECTED_COMPONENTS="$DEMO_COMPONENTS"
+                SELECTED_COMPONENTS="$SELECTED_COMPONENTS $DEMO_COMPONENTS"
```

Comment:
ok, so this virtually means, that commercial components (atm monitor-client) will not be installed with --demo. is that what we want?

Reply:
That is a good question. On the conceptual side, you are right in questioning this as "demo" should include all the packages (iow, include commercial packages). But on the implementation side, that would mean that "demo" cannot be used out of Hosted Mender (the JWT would be required always).

That is the reason for which I decided to set it up separately.

Reply:
since we are discussing today the onboarding release testing, perhaps we shall also briefly discuss that?

Reply:
It is related, yes, so we can discuss it. I have an strong opinion though, in that the on-boarding script shall work for OS...